### ISSUE TITLE
Pain in hallux

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11179]
**Preferred term label:**
Pain in hallux

**Synonyms**
First toe pain; great toe pain; big toe pain; pain along the top, bottom, or sides of the first toe at rest or with range of motion of the toe

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C1644184

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sesamoid injuries of the hallux
Gout, idiopathic
Hallux valgus, adult
Flexor hallucis longus (FHL) tendon dysfunction

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001401
- label: Hallux pain
- parents: [Toe pain (HP:0030841)](https://hpo.jax.org/app/browse/term/HP:0030841)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the big toe (hallux).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11179
- synonyms: Pain in hallux(NONE)|Big toe pain(NONE)|Great toe pain(NONE)|First toe pain(NONE)
